### PR TITLE
New release of Menhir.

### PR DESCRIPTION
**Pull request by fpottier**
Thanks!
François.

### CODE REVIEW

File: packages/menhir/menhir.20170509/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "http://gallium.inria.fr/~fpottier/menhir/menhir-20170509.tar.gz"
```
Comment (yallop) [inline packages/menhir/menhir.20170509/url]: This URL is currently returning 404.  Is it just a temporary problem, @fpottier?

**Comment by fpottier** [comment]
Hi Jeremy,

Le 10/05/2017 20:34, yallop a écrit :
> This URL is currently returning 404. Is it just a temporary problem,
> @fpottier <https://github.com/fpottier>?

Our server was powered down for a few hours today,
but I believe it is back up now.

-- 
François Pottier
francois.pottier@inria.fr
http://gallium.inria.fr/~fpottier/

File: packages/menhir/menhir.20170509/url (hunk 1 above)
Comment (jordwalke) [inline packages/menhir/menhir.20170509/url]: This is causing problems for our offline mirrors that we create from opam, though I don't know if it also causes problems for direct use of opam.

**Comment by jordwalke** [comment]
It's still "Not Found".

http://gallium.inria.fr/~fpottier/menhir/menhir-20170509.tar.gz

**Comment by jonludlam** [comment]
This is also causing us problems.

**Comment by fpottier** [comment]
Le 11/05/2017 05:58, Jordan W a écrit :
> It's still "Not Found".

Our Web server is down and won't restart. Looking into it;
sorry for the trouble.

-- 
François Pottier
francois.pottier@inria.fr
http://gallium.inria.fr/~fpottier/

**Pull request merged**